### ISSUE TITLE
Provide start/end functionality to profiling

### ISSUE THREAD

**Issue by atian25** [medikoo/memoizee#37]

**Comment by medikoo** [comment]
Yes, it will, it immediately attaches the logger, and from now on data usage is being collected

**Comment by atian25** [comment]
it will be nice to provide `profile.start()`, so we can open/close this feature by a config online.

**Comment by medikoo** [comment]
@atian25 It sounds reasonable, I'll take that into consideration for next version

**Comment by atian25** [comment]
any news for this?

**Comment by medikoo** [comment]
@atian25 at this point I don't have time to handle that. It's likely to happen may-june.

**Comment by atian25** [comment]
ok, I'm trying to use lazy require to workaround this.

**Comment by medikoo** [comment]
Let's keep it open, I'd like to address that
